src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -188,10 +197,14 @@ public void copyFile(File file) {
             ZipUtil.pack(file, tmp);
             byte[] archive = IOUtils.toByteArray(new FileInputStream(tmp));
 
-            addShellStep(String.format(
-                    "base64 --decode << ENDOFFILE > archive.zip && unzip -o archive.zip \n%s\nENDOFFILE",
-                    new String(Base64.encodeBase64Chunked(archive))
-            ));
+            if (SystemUtils.IS_OS_WINDOWS) {
+                addBatchStep("xcopy " + file.getAbsolutePath() + " %cd% /E");
```

Comment:
:bug: IIUC this will not work at all for a `RemoteJenkinsController`—hence the complex trick for Unix. If you do not want to implement it correctly for Windows now, OK, but make sure this method throws a meaningful error when a remote controller is in use.